packages/api/emails/verifyHcEmail/html.pug

```diff
@@ -0,0 +1,22 @@
+div.main
+  img(src=`${process.env.APP_URL}/email/header.jpg` alt='header' style='width:100%; height:auto;')
```

Comment:
We might want to add this to the `locals` object similar to how `name` is passed through for consistency in how we pass variables through to the email template.

Reply:
complete!